package/cfg/k3s-cis-1.9/config.yaml

```
@@ -42,7 +42,7 @@ node:
     bins:
       - containerd
     confs:
-      - /var/lib/rancher/k3s/agent/kubelet.kubeconfig
+      - /var/lib/rancher/k3s/agent/etc/kubelet.conf.d/00-k3s-defaults.conf
     defaultkubeconfig: /var/lib/rancher/k3s/agent/kubelet.kubeconfig
```

```suggestion
    defaultkubeconfig: /var/lib/rancher/k3s/agent/etc/kubelet.conf.d/00-k3s-defaults.conf
```
Should the `defaultkubeconfig` be changed too ?

---

No, `confs` are different from `kubeconfig` and the `defaultkubeconfig`. In k3s, the kubelet.kubeconfig was incorrect marked as both types

---

Got you ! Thanks for confirming.